src/index.ts

```diff
@@ -29,7 +29,10 @@ function defaultSerializeQueryArgs(args: any, endpoint: string) {
   return `${endpoint}/${JSON.stringify(args)}`;
 }
 
-const IS_DEV = () => process?.env?.NODE_ENV === 'development';
+const IS_DEV = () => {
+  return true;
+  return process?.env?.NODE_ENV === 'development';
```

Comment:
For some reason this throws `process is undefined` randomly when running the React sample app.

Reply:
Can you try
```js
const IS_DEV = () => typeof process !== "undefined" && process.env.NODE_ENV === 'development';
```
?